source/sdk/math.ooc

```diff
@@ -74,22 +70,8 @@ extend Int {
 		result % divisor
 	}
 	clamp: func (floor: This, ceiling: This) -> This {
-		if (this > ceiling)
-			ceiling
-		else if (this < floor)
-			floor
-		else
-			this
+		this > ceiling ? ceiling : (this < floor ? floor : this)
```

Comment:
It would be slower, I guess, but `this maximum(floor) minimum(ceiling)` should also work (if `maximum` and `minimum` are methods and not static functions).